package.json

```diff
@@ -29,11 +29,12 @@
     "docs:build": "npx typedoc",
     "docs:build:watch": "npm run docs:build -- --watch",
     "docs:serve": "PORT=4000 npx serve --no-clipboard docs",
-    "test": "npx jest",
-    "test:ci": "npm run test -- --ci --verbose false",
-    "test:watch": "npm run test -- --watchAll",
-    "test:e2e": "npm run test -- --config __tests__/e2e/jest.e2e-config.js",
-    "test:e2e:watch": "npm run test -- --config __tests__/e2e/jest.e2e-config.js --watchAll",
+    "test:unit": "npx jest --ci",
```

Comment:
Shouldn't we have separate commands to run the tests in a local environment and in the [ci](https://jestjs.io/es-ES/docs/cli#--ci)?

Reply:
Done!